appinventor/components-ios/src/Form.swift

```diff
@@ -349,6 +371,8 @@ let kMinimumToastWait = 10.0
     TitleVisible = true
     ShowListsAsJson = true
     ScreenOrientation = "unspecified"
+    BigDefaultText = false
+    HighContrast = false
```

Comment:
Since properties are typically set in order by the Scheme code, we should sort these accordingly among the above code.